nim_status/lib/account.nim

```diff
@@ -0,0 +1,62 @@
+import
```

Comment:
Wouldn't it make more sense for this code to be added to the `nim-eth` library or even a stand-alone nim-bip32 library?